WinUIGallery/Samples/ControlPages/Design/IconographyPage.xaml

```diff
@@ -72,6 +72,16 @@
                         Text="{Binding Name}"
                         TextTrimming="CharacterEllipsis"
                         TextWrapping="NoWrap" />
+
+                    <InfoBadge 
```

Comment:
Do we actually need this? Just thinking out loud: one would search for an icon, and could then use the `Symbol` if it would be available. What would a visual indicator add?

Reply:
Also thinking out loud: Adding `SymbolIcon` as tag should help filtering. 🧐

Reply:
> Do we actually need this?

I think yes. So that you don't have to open the individual icons to know if they allow you to use a `SymbolIcon`.



> Adding `SymbolIcon` as tag to the corresponding icons.

Could do in a separate PR

Reply:
But I think that's my point; why would someone be searching for icons that are part of symbolicons?

Wouldn't they be searching for the right icon, and if SymbolIcon is available, they could use that, and if not, use FontIcon?

Reply:
I've decreased the size of that `InfoBadge` from 8 to 6 and changed the opacity to 0.4 to minimize its footprint.

Reply:
I agree with @niels9001, I see little value in being able to see this quickly. If this was a criteria, we would also need to allow to hide all non symbol icons via a filter. Since we do not do this, I think its better to not highlight this.

Reply:
Removed.